zerver/views/user_settings.py

```diff
@@ -55,13 +55,12 @@ def confirm_email_change(request: HttpRequest, confirmation_key: str) -> HttpRes
 
 @human_users_only
 @has_request_variables
-def json_change_ui_settings(request, user_profile,
-                            autoscroll_forever=REQ(validator=check_bool,
-                                                   default=None),
-                            default_desktop_notifications=REQ(validator=check_bool,
-                                                              default=None)):
-    # type: (HttpRequest, UserProfile, Optional[bool], Optional[bool]) -> HttpResponse
-
+def json_change_ui_settings(
```

Comment:
I agree with the set-back indent for the long lines with `REQ`, but for consistency perhaps keep the `request` and `user_profile` on the same line in this change?